### PR TITLE
Add documentation on S3 scratch bucket

### PR DESCRIPTION
**Pull request by scottyhq**
Addresses #68

### CODE REVIEW

File: book/how_tos/data_access/CryoCloudScratchBucket.ipynb [hunk 1]
```diff
@@ -0,0 +1,1480 @@
...
+    "## Access the scratch bucket\n",
+    "\n",
+    "The CryoCloud scratch bucket is hosted at `s3://nasa-cryo-scratch`. CryoCloud JupyterHub automatically sets an environment variable `SCRATCH_BUCKET` that appends a suffix to the s3 url with your GitHub username. This is intended to keep track of file ownership, stay organized, and prevent users from overwriting data!\n",
+    "\n",
+    "```{warning}\n",
+    "Everyone has full access to the scratch bucket, so be careful not to overwrite data from other users when uploading files. Also, any data you put there will be deleted 7 days after it is uploaded\n",
+    "```\n",
+    "\n",
+    "```{hint}\n",
+    "If you need permamnent storage refer to [These Docs](./Instructions_for_configuring_AWS_S3_bucket.ipynb)\n",
```
Comment (tsnow03) [inline book/how_tos/data_access/CryoCloudScratchBucket.ipynb:36]: If you need permanent S3 bucket storage

File: book/how_tos/data_access/CryoCloudScratchBucket.ipynb [hunk 2]
```diff
@@ -0,0 +1,1480 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "eced7ea8-79e7-4bcf-a723-3221eec6018e",
+   "metadata": {
+    "user_expressions": []
+   },
+   "source": [
+    "## Accessing Data\n",
+    "\n",
+    "Some software packages allow you to stream data directly from S3 Buckets. But you can always pull objects from S3 and work with local file paths. For file formats that are not Cloud Optimized (like HDF!) this often gives the best performance. \n",
```
Comment (tsnow03) [inline book/how_tos/data_access/CryoCloudScratchBucket.ipynb:291]: For file formats (like HDF!) that are not cloud-optimized this often gives the best performance.

I confused this as saying HDF was cloud optimized at first

File: book/how_tos/data_access/CryoCloudScratchBucket.ipynb [hunk 3]
```diff
@@ -0,0 +1,1480 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "10e49cc4-38fd-4c46-816c-f09c0070529b",
+   "metadata": {
+    "user_expressions": []
+   },
+   "source": [
+    "## Cloud-optimized formats\n",
+    "\n",
+    "Other formats like [COG](https://www.cogeo.org), [ZARR](https://zarr.readthedocs.io/en/stable/), [Parquet](https://parquet.apache.org) are 'Cloud-optimized' and allow for very efficient streaming directly from S3. In other words, you do not need to download entire files for best performance and can easily read subsets."
```
Comment (tsnow03) [inline book/how_tos/data_access/CryoCloudScratchBucket.ipynb:1289]: subsets of the data.

**Review by tsnow03 — APPROVED** [review]
I added 3 suggestions, but everything else looks great! Thanks Scott!

**Comment by jomey** [comment]
Not sure if this is a requirement here, but should the jupyterbook be committed as rendered? More of a curious question.

**Comment by jomey** [comment]
Also along the lines of 'rendered version', would it be good to have this example not tailored to your GitHub username and use a generic one?
Mainly wondering if someone just wants to test this notebook and walks through the steps that this person might overwrite your personal bucket :smile:

**Comment by scottyhq** [comment]
> Not sure if this is a requirement here, but should the notebook be committed as rendered? More of a curious question.

https://github.com/CryoInTheCloud/CryoCloudWebsite/blob/79d39c9e90c158b39feaf3134659cc377f794c07/book/_config.yml#L36-L37

Based on that setting the notebook must be rendered. Probably worth opening another issue to discuss simplifying github actions borrowed from the uw hackweeks.

**Comment by scottyhq** [comment]
> would it be good to have this example not tailored to your GitHub username and use a generic one? 

Good point! I'll use [`octocat`](https://github.com/octocat)

**Comment by scottyhq** [comment]
Thanks for the review @tsnow03 and @jomey I think this is ready to merge!

**Review by jomey — APPROVED** [review]

**Pull request merged**